test/helpers/data.go

```diff
@@ -15,18 +15,20 @@ const (
 
 var (
 	r        *rand.Rand
-	rndMutex *sync.Mutex
-	once     sync.Once
+	rndMutex sync.Mutex
 )
 
-func initSeed() {
-	seed := time.Now().UTC().UnixNano()
-	r = rand.New(rand.NewSource(seed))
-	rndMutex = &sync.Mutex{}
+func init() {
+	r = rand.New(rand.NewSource(time.Now().UTC().UnixNano()))
```

Comment:
I'd just `rand.Seed` here and save the mutex

Reply:
Done.